.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -36,6 +36,13 @@ These questions are the first thing we need to know to understand the context.
 
 **Apache Airflow version**:
 
+**Apache Airflow Provider versions** (please include all providers that are relevant to your bug):
+
```

Comment:
Actually better advice will be 'airflow providers list' command output

Reply:
TIL. That said, I think I prefer the output from pip as it doesn't have the description (assuming folks will copy/paste most likely). Thoughts?

Reply:
The one difference is that providers list might contain more than community providers. Those are all registered ones which is discovered by scanning the entrypoints.

For issues in community we indeed care more about the community ones though. So i think both are fine and agree that description is not really needed in this case so `pip freeze` will do its job well.

One other difference that airflow providers works also when you use INSTALL_PROVIDERS_FROM_SOURCES flag and providers are discovered from local sources rather than from packages (but this is not our case here)